src/Element/NodeElement.php

```diff
@@ -237,6 +237,20 @@ public function selectOption($option, $multiple = false)
         $this->getDriver()->selectOption($this->getXpath(), $opt->getValue(), $multiple);
     }
 
+    /**
+     * Returns that the option is in the element or not.
+     * 
+     * @param string $option
+     * @return Boolean
+     */
+    public function hasOption($option){
+        if ('select' !== $this->getTagName()) {
+            return;
```

Comment:
I writed this because if the element is not a select tag, It is not logical to return a boolean. That would be nice if I throw an exception instead of return null? Or editing the documentation of method to `@return Boolean|null`?

Reply:
IMO it's better to throw an exception